core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -457,8 +446,10 @@ def xpub_title(i: int) -> str:
             result = await ctx_wait(
                 RustLayout(
                     trezorui2.show_address_details(
+                        title=title,
                         address=address if address_qr is None else address_qr,
                         case_sensitive=case_sensitive,
+                        subtitle=subtitle,
```

Comment:
`title` and `subtitle` are little misleading names here, as they are not displayed on the same page.

`AddressDetails::new` has it as `qr_title` and `details_title`, we might want to make it so in `trezorui2.show_address_details` as well

Reply:
9fda87ed8bbd0d362d51c4b45e1d18edfd41419b